### PR TITLE
bump-up-to-v0.4.0-beta.1

### PR DESCRIPTION
**Pull request by tuommaki**

### CODE REVIEW

File: helm/cluster-api-bootstrap-provider-kubeadm/Chart.yaml [hunk 1]
```diff
@@ -3,4 +3,6 @@ name: cluster-api-bootstrap-provider-kubeadm
 description: Cluster API bootstrap provider Kubeadm (CABPK) is a component of Cluster API that is responsible of generating a cloud-init script to turn a Machine into a Kubernetes Node; this implementation uses kubeadm for kubernetes bootstrap.
 home: https://github.com/giantswarm/cluster-api-bootstrap-provider-kubeadm-app
 version: [[ .Version ]]
-appVersion: 0.3.19-gs1
+appVersion: 0.4.0
```
Comment (fiunchinho) [inline helm/cluster-api-bootstrap-provider-kubeadm/Chart.yaml:6]: I've seen `[[ .AppVersion ]]` in other places. I think that's the one we should use?

File: helm/cluster-api-bootstrap-provider-kubeadm/templates/service.yaml [hunk 2]
```diff
@@ -11,10 +11,12 @@ metadata:
   namespace: {{ include "resource.default.namespace" . }}
```
Comment (fiunchinho) [inline helm/cluster-api-bootstrap-provider-kubeadm/templates/service.yaml:11]: I see these annotations in other `Services`, I'd add them here for alignment sake
```
    prometheus.io/scrape: "true"
    giantswarm.io/monitoring: "true"
    giantswarm.io/monitoring-port: "{{ .Values.ports.metrics }}"
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Updated cluster-api to v0.3.19.
+### Changed
+
+- Bumped `cluster-api` to version `v0.4.0`.
+- Removed dedicated container for the webhook.
```
Comment (fiunchinho) [inline CHANGELOG.md:13]: Also 
```
- Using a static name for the app manifests, so that they don't change when releasing a new version.
```

**Comment by fiunchinho** [comment]
I think you need to drop the `--webhook.` part from the `dnsName` in the `Certificate`.

File: helm/cluster-api-bootstrap-provider-kubeadm/Chart.yaml (hunk 1 above)
Comment (tuommaki) [reply helm/cluster-api-bootstrap-provider-kubeadm/Chart.yaml]: Changed

File: helm/cluster-api-bootstrap-provider-kubeadm/templates/service.yaml (hunk 2 above)
Comment (tuommaki) [reply helm/cluster-api-bootstrap-provider-kubeadm/templates/service.yaml:13]: Added

File: CHANGELOG.md (hunk 3 above)
Comment (tuommaki) [reply CHANGELOG.md:13]: Added

**Comment by fiunchinho** [comment]
Superseeded by https://github.com/giantswarm/cluster-api-app

**Pull request closed without merging**